test/mocha/new.js

```diff
@@ -2,7 +2,7 @@ var assert = require("assert");
 var uglify = require("../../");
 
 describe("New", function() {
-    it("Should attach callback parens after some tokens", function() {
+    it("Should add trailing parentheses for new expressions with zero arguments in beautify mode", function() {
```

Comment:
If all inputs in this mocha test already exist in a compress test already using `expect_exact` then no need.